test/hooks_test.dart

```diff
@@ -666,6 +666,130 @@ main() {
         });
       });
     });
+
+    group('useMemo -', () {
+      ReactDartFunctionComponentFactoryProxy UseMemoTest;
+      StateHook<int> num;
```

Comment:
#nit probably shouldn't use `num` as a variable name - since its a reserved word / primitive type in Dart's type system.